pep-0501.txt

```diff
@@ -501,11 +653,12 @@ a creating a new kind of object for later consumption by interpolation
 functions. Creating a rich descriptive object with a useful default renderer
 made it much easier to support customisation of the semantics of interpolation.
 
-Building atop PEP 498, rather than competing with it
-----------------------------------------------------
+Building atop PEP 701 rather than competing with PEP 498
+--------------------------------------------------------
```

Comment:
```suggestion
Building atop f-strings rather than replacing them
--------------------------------------------------
```

It doesn't really make sense to "compete" with a PEP that was accepted and implemented nearly a decade ago. Furthermore, use the human-readible, modern feature names rather than historic PEP numbers, and avoid any confusion as to the relationship of the two other PEPs named.